README.md

```diff
@@ -1,23 +1,26 @@
-# SeqAn3 App Template
+# SeqAn3 App Template [![build status][1]][2] [![codecov][3]][4]
 
-[![Build Status](https://github.com/seqan/app-template/workflows/App%20CI/badge.svg)](https://github.com/seqan/app-template/actions?query=branch%3Amaster+workflow%3A%22App+CI%22)
+[1]: https://img.shields.io/github/workflow/status/seqan/app-template/CI%20on%20Linux/master?style=flat&logo=github&label=App-Template%20CI
+[2]: https://github.com/seqan/app-template/actions?query=branch%3Amaster
+[3]: https://codecov.io/gh/seqan/app-template/branch/master/graph/badge.svg?token=V82JRCXF0K
+[4]: https://codecov.io/gh/seqan/app-template
```

Comment:
nice :)

Reply:
can you investigate if we can add comments after / before the links to indicate whether they are links or image urls?